cpp/src/parquet/column_reader_benchmark.cc

```diff
@@ -219,5 +251,19 @@ BENCHMARK(RecordReaderReadRecords)
     ->Args({2, 1000, true})
     ->Args({2, 1000, false});
 
+BENCHMARK(RecordReaderReadAndSkipRecords)
+    ->ArgNames({"Repetition", "BatchSize"})
```

Comment:
nit: add name for levels_per_page?

Reply:
BTW, should the batch size be power of two which is a common case?

Reply:
done, thanks for catching this! I feel here probably magnitude matters more than power-of-two (or not power-of-two might be better) for testing purpose.